ui/src/components/header/ReadOnly.tsx

```diff
@@ -32,8 +32,20 @@ export default function ReadOnly() {
         config.storage?.type || 'unknown'
       )} Storage`}
     >
-      {StorageIcon && (
-        <StorageIcon className="h-4 w-4 fill-gray-200" aria-hidden="true" />
+      {config.storage?.type == 'git' && (
+        <span
+          className="max-w-32 truncate font-medium"
```

Comment:
```suggestion
          className="nightwind-prevent text-violet-200 max-w-32 truncate font-medium"
```

wdyt?

![CleanShot 2024-01-17 at 16 47 03@2x](https://github.com/flipt-io/flipt/assets/209477/07fce275-fd26-435b-9d3a-3c48d1d1ce58)

Reply:
it looks great!